lib/dai-plugin-mcd/src/utils.js

```diff
@@ -11,3 +11,15 @@ export function getIlkForCurrency(currency, convertToBytes = true) {
   assert(ilk, "ilk can't be blank");
   return convertToBytes ? stringToBytes32(ilk, false) : ilk;
 }
+
+export function getSpotContractNameForCurrency(currency){
+	return 'MCD_SPOT_' + currency.symbol;
```

Comment:
your indentation seems to be off in this file and the spec file (is your editor saving tabs instead of spaces)?